erigon-lib/common/metrics/block_metrics.go

```diff
@@ -31,6 +31,8 @@ var (
 	BlockConsumerPreExecutionDelay   = metrics.NewSummary(`block_consumer_delay{type="pre_execution"}`)
 	BlockConsumerPostExecutionDelay  = metrics.NewSummary(`block_consumer_delay{type="post_execution"}`)
 	BlockProducerProductionDelay     = metrics.NewSummary(`block_producer_delay{type="production"}`)
+	ObservedWaypointCheckpointLength = metrics.NewSummary(`observed_waypoint_length{type="checkpoint"}`)
+	ObservedWaypointMilestoneLength  = metrics.NewSummary(`observed_waypoint_length{type="milestone"}`)
```

Comment:
do we need these to be `summary`-ies or would not a `gauge` metric type be more suitable?